pydm/widgets/image.py

```diff
@@ -344,8 +346,6 @@ def image_width_changed(self, new_width):
         new_width : int
             The new image width
         """
-        if new_width is None:
```

Comment:
Why was this removed?

Reply:
The slot only expects an int value. If new_width is None, an exception will be raised, so there's no need for a None check here.

Reply:
We should never allow the code to drag down the whole application.
This code can also be invoked from command line and one can end up sending `None` down the pipe and crash if we don't protect with the proper check.
Please add again the line that was removed.